app/posts/apply-for-teacher-training/2020-02-11-training-with-a-disability-iteration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Giving details about disability (iteration)
+description:
+date: 2020-02-11
+---
+This design is [an iteration of the question as designed for the pilot](/apply-for-teacher-training/training-with-a-disability). This question was removed prior to launch due to a lack of confidence in how we were asking a sensitive and possibly contentious question.
+
+## User needs
+
+As a candidate
+: I need to inform a provider of a disability, health issue or access need I have,
+: so that reasonable adjustments can be made should I be offered an interview and/or begin training with them.
+
+As a candidate
+: I need reassurance that disclosing a disability won’t affect my application, either due to unconscious bias or discrimination,
+: so that I can feel comfortable disclosing this information.
+
+As a provider
+: I need to know if a candidate needs extra support,
+: so that I can make any reasonable adjustments.
+
+## Hypotheses
+
+### Using ‘extra assistance’ in the title
+
+Using the word disability in the title excluded candidates that have (short-term) physical or mental health conditions, or a condition they don’t regard as a disability.
+
+If we don’t explicitly use the word disability, but instead use the term ‘extra assistance’, then candidates with access needs will feel more compelled to provide details of their condition(s). We’ll know this works when more candidates provide relevant details of their particular needs.
```

Comment:
```suggestion
If we use say ‘extra assistance’ instead of mentioning ‘disability’
Then candidates with access needs will be more compelled to enter their condition(s)
We’ll know this works when:
- more candidates provide relevant details of their particular needs
```